Author: Eric Pruitt, https://github.com/ericpruitt/
Description: Ensure that mouse actions on the window title area of the status
bar are only registered if they happen in the top half of the bar. This is to
reduce the amount of accidental window closures caused by attempts to
middle-mouse browser tabs.

diff --git a/dwm.c b/dwm.c
index 169adcb..8518f8c 100644
--- a/dwm.c
+++ b/dwm.c
@@ -434,6 +434,8 @@ buttonpress(XEvent *e) {
 			click = ClkLtSymbol;
 		else if(ev->x > selmon->ww - TEXTW(stext))
 			click = ClkStatusText;
+		else if (ev->y > bh / 2)
+			return;
 		else
 			click = ClkWinTitle;
 	}
